Unity-Technologies/com.unity.mobile.logcat
# Autorun will choose correct device when launching application

## **Please read and consider what information you want to pass on to people reviewing this PR**

### Purpose of PR
**Type of change:**

* **Description of the feature**

Logcat was connecting to the wrong device, when there were two or more devices connected to PC. The issue was because Unity didn't provide any info on which device it runs the application.

OnPostrocessLaunch callback was introduced in this PR - https://github.cds.internal.unity3d.com/unity/unity/pull/56719

### Checklist for PR maker
- [ ] Have you added a backport label? (if needed)
- [ ] Have you updated the Changelog? _Each package has a `CHANGELOG.md` file_
- [ ] **Have you added or updated the Documentation to your PR?** _When you add a new feature, change a property name, or change the behaviour of a feature, it's best practice to include related documentation changes in the same PR_
---
### Testing status
* **Existing or new automation tests - what automation was added, changed**
N/A
* **Detailed description of what testing was done, what projects were run**
  * Connect two devices to PC
  * Build & Run on device A, see that logcat connects to device A
  * Build & Run on device B, see that logcat connects to device B

* **What environment did you test on?**
Windows, Android 10 and Android 14 connected at the same time.

### Note to guardian
_Changeset to use, if extra changesets on top. Any other notes to pass along_

### Comments to reviewers
_Notes for the reviewers you have assigned_
